packages/vue-components/src/Retriever.vue

```diff
@@ -58,13 +58,15 @@ export default {
       if (this.hasFetchedCopy) {
         return;
       }
-      jQuery.get(this.srcWithoutHash)
-        .done((response) => {
-          let result = response;
+      fetch(this.srcWithoutHash)
+        .then(response => response.text())
+        .then((htmlText) => {
+          let result = htmlText;
           if (this.hash) {
-            const tempDom = jQuery('<temp>').append(jQuery.parseHTML(result));
-            const appContainer = jQuery(`#${this.hash}`, tempDom);
-            result = appContainer.html();
+            const htmlResult = document.implementation.createHTMLDocument('');
+            htmlResult.body.innerHTML = result;
+            const appContainer = htmlResult.querySelector(`#${this.hash}`);
+            result = appContainer.innerHTML;
```

Comment:
This part basically puts the html into a temp tag so that the hash can be queried for. 

So in replacement, a fake html document is created, html is loaded inside the body and the hash is queried.